query-engine/connectors/sql-query-connector/src/limit.rs

```diff
@@ -0,0 +1,31 @@
+use crate::{model_extensions::*, Context};
+use quaint::ast::*;
+use query_structure::*;
+
+pub(crate) fn wrap_with_limit_subquery_if_needed<'a>(
```

Comment:
ℹ️ Using the same logic to create a subquery for updates and deletes => centralized that logic here. 

Happy to learn whether we have a better place for such shared logic. :)